### PR TITLE
More guidance for exercises

### PR DESCRIPTION
**Pull request by hfrick**
Rendering this introduces a lot of changes, so I haven't pushed those.

### CODE REVIEW

File: classwork/05-classwork.qmd [hunk 1]
```diff
@@ -115,6 +115,7 @@ Create a `recipe()` for the on-goal data to :
 
 -   create dummy variables
 -   remove zero-variance variables
+-   any additional steps needed for your chosen model from Day 1
```
Comment (DavisVaughan) [inline classwork/05-classwork.qmd:118]: I dont actually think we should include this. I _dont_ think people will be using a different model than us. I expect everyone to be using the same model at this point

**Comment by juliasilge** [comment]
We are going to need to do a round through the classwork `.qmd` files and harmonize with the changes we are making today/tomorrow. @hfrick are you able to do that before/as you travel, or should one of us take it?

**Comment by hfrick** [comment]
I can do that during travel 👍

**Comment by hfrick** [comment]
I'm gonna close this PR and open a fresh one, once the slides have been updated

**Pull request closed without merging**